src/ray/object_manager/object_directory.cc

```diff
@@ -36,6 +39,15 @@ void UpdateObjectLocations(const std::vector<ObjectTableDataT> &location_history
     } else {
       client_ids->erase(client_id);
     }
+    if (object_table_data.inline_object_flag) {
```

Comment:
Does it make sense to still update the client list if `inline_object_flag` is true? I think it is clearer not to do so, since only the first client to create the object will actually be added as a location.